src/store/modules/job/actions.ts

```diff
@@ -248,6 +248,57 @@ const actions: ActionTree<JobState, RootState> = {
       showToast(translate("Something went wrong"));
     })
   },
+  async fetchMiscellaneousJobs({ commit, dispatch, state }, payload){
+    const params = {
+      "inputFields": {
+        "enumTypeId": "MISC_SYS_JOB",
+        "statusId": ["SERVICE_PENDING", "SERVICE_DRAFT"],
```

Comment:
We are fetching pending and draft jobs, I think we should handle them separately on UI. We cannot skip/cancel draft jobs, also how these could distinguished on UI.

Reply:
Got it sir. Fixed it.